android/engine/build.gradle

```diff
@@ -51,6 +83,23 @@ dependencies {
     implementation 'androidx.security:security-crypto:1.1.0-alpha03'
     implementation 'joda-time:joda-time:2.10.6'
     implementation 'androidx.paging:paging-runtime-ktx:3.0.1'
+    testImplementation group: 'org.mockito', name: 'mockito-core', version: '2.19.0'
```

Comment:
I don't think we need this dependency as we are using `Mockk`

Reply:
@ndegwamartin. Also will refactor